editor/src/components/canvas/controls/classname-select.tsx

```diff
@@ -66,12 +157,160 @@ export const ClassNameSelect: React.FunctionComponent = betterReactMemo('ClassNa
           value: name,
         }))
 
+  const colourStyles: StylesConfig = {
```

Comment:
Is this something that should be wrapped in a `React.useMemo`?

Reply:
better yet, could this be moved to the file root?

Reply:
It uses `theme` so it would have to become a function that is wrapped in a `useMemo` still.